nsv13/modules/credits/episode_name.dm

```diff
@@ -0,0 +1,497 @@
+/datum/episode_name
+	var/thename = ""
+	var/reason = "Default episode name." //Explanation on why this episode name fits this round. For the admin panel.
+	var/weight = 100 //50 will have 50% the chance of being picked. 200 will have 200% the chance of being picked, etc. Relative to other names, not total (just the default names already total 700%)
+	var/rare = FALSE //If set to true and this episode name is picked, the current round is considered "not a rerun" for client preferences.
+
+/datum/episode_name/rare
+	rare = TRUE
+
+/datum/episode_name/New(var/thename, var/reason, var/weight)
+	if(!thename)
+		return
+	src.thename = thename
+	if(reason)
+		src.reason = reason
+	if(weight)
+		src.weight = weight
+
+	switch(rand(1,100))
+		if(0 to 5)
+			thename += ": PART I"
+		if(6 to 10)
+			thename += ": PART II"
+		if(11 to 12)
+			thename += ": PART III"
+		if(13)
+			thename += ": NOW IN 3D"
+		if(14)
+			thename += ": ON ICE!"
+		if(15)
+			thename += ": THE SEASON FINALE"
+		if(16 to 40)
+			//if(score["time"] > 60 * 60 * 3) //3 hours
+			thename += ": THE FEATURE LENGTH PRESENTATION"
+		if(41 to 65)
+			//if(score["time"] > 0 && score["time"] < 60 * 30) //30 min
+			thename += ": ABRIDGED"
+		else
+			for(var/client/C in GLOB.clients)
+				if(C.key && (C.key == "pomf123" || C.key == "exadv1"))
```

Comment:
```suggestion
				if(C.key && (C.key == "kmc2000" || C.key == "exadv1"))
```

Reply:
this will all likely get changed to a config. We will have MULTIPLE config files which link to other config files if we have to!